hoprd/rest-api/src/lib.rs

```diff
@@ -249,6 +252,7 @@ async fn build_api(
                 .route("/node/peers", get(node::peers))
                 .route("/node/entryNodes", get(node::entry_nodes))
                 .route("/node/metrics", get(node::metrics))
+                .route("/session/client", get(session::create_client))
```

Comment:
Since the semantics of this endpoint is that it creates a session it should use `POST` instead.
```suggestion
                .route("/session", post(session::create_client))
```